app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor i not vaccinated")
+
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Visitors vaccine is expired")
+
+        if not visitor["wearing_a_mask"]:
+            raise NotWearingMaskError("Visitor did not wear a mask")

```

Comment:
Code Style: Your error message is not descriptive enough. Try to provide more information about the error. For example, you could say 'The visitor is not wearing a mask. Wearing a mask is required to enter the cafe.' instead of 'Visitor did not wear a mask'.